### ISSUE TITLE
use built in `locate-command` in dired-list-locate

### ISSUE THREAD

**Issue by oneness** [Fuco1/dired-hacks#183]
Currently dired-list-locate fn uses static `locate ` command to list located files in dired buffer.  I recommend using `locate-command` built-in custom var so user can customize which locate command gets used. For example:
```
(setq locate-command (if (string= system-type "darwin") "mdfind" "fd"))
```
Then, user of Linux could use `fd` or `mdfind` (osx). 

I will do a PR shortly to address this. 

BTW, Thanks for this great dired hacks especially dired-list and friends.

**Comment by oneness** [comment]
It exists in Emacs 27 onward.  The latest update to my PR does exactly what you suggested.

**Comment by Fuco1** [comment]
Was fixed in #184
